compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -510,10 +513,61 @@ class ReplDriver(settings: Array[String],
         state
       }
 
+    case Require(path) =>
+      out.println(":require has been deprecated and replaced with :jar. Please use :jar")
```

Comment:
nitpick: since we're deprecating it, it should technically still redirect to `:jar`.
Alternatively, we can plain drop it with:
```suggestion
      out.println(":require is no longer supported, but has been replaced with :jar. Please use :jar")
```